test/jdk/javax/swing/JSpinner/8223788/JSpinnerButtonFocusTest.java

```diff
@@ -85,19 +94,11 @@ public static void main(String args[]) throws Exception {
 
                     frame.setAlwaysOnTop(true);
                     frame.pack();
+                    frame.setLocationRelativeTo(null);
                     frame.setVisible(true);
                 });
                 robot.waitForIdle();
-
-                editor1.getTextField().addFocusListener(new FocusAdapter() {
-                    @Override
-                    public void focusGained(FocusEvent e) {
-                        super.focusGained(e);
-                        robot.keyPress(KeyEvent.VK_TAB);
-                        robot.keyRelease(KeyEvent.VK_TAB);
-                        latch1.countDown();
-                    }
-                });
+                robot.delay(1000);
 
```

Comment:
This is on unedited line: Ln#107, 15 minutes of await time looks really long. I think 1 or 2 mins would be more than sufficient ?

```
 if (!latch1.await(15, TimeUnit.MINUTES)) {
                    throw new RuntimeException(LF.getClassName() +
                            ": Timeout waiting for editor1 to gain focus.");
}
```

Reply:
ok..i noticed it but did not change it as it was not contributing to the failure